crates/youki/src/commands/mod.rs

```diff
@@ -32,3 +35,30 @@ fn load_container<P: AsRef<Path>>(root_path: P, container_id: &str) -> Result<Co
     Container::load(container_root)
         .with_context(|| format!("could not load state for container {}", container_id))
 }
+
+fn create_cgroup_manager<P: AsRef<Path>>(
+    root_path: P,
+    container_id: &str,
+) -> Result<Box<dyn CgroupManager>> {
+    let container = load_container(root_path, &container_id)?;
+    let config_absolute_path = container.root.join("config.json");
+    log::debug!("load spec from {:?}", config_absolute_path);
+    let spec = oci_spec::runtime::Spec::load(config_absolute_path)?;
+    log::debug!("spec: {:?}", spec);
+    let cgroups_path = utils::get_cgroup_path(
+        spec.linux()
+            .as_ref()
+            .context("no linux in spec")?
+            .cgroups_path(),
+        container.id(),
+    );
```

Comment:
You can use container.spec() for this. We do not save the whole runtime spec during container creation anymore, only the parts that will be needed later on. This is the reason why trying to load config.json fails.